lib/log.js

```diff
@@ -18,12 +18,21 @@ export function createLogger(details) {
 
 export function fmtLog() {
   let args = Array.prototype.slice.call(arguments);
-  return '[taskcluster] ' + util.format.apply(this, args) + '\r\n';
+  let ts = new Date().toISOString().replace('T', ' ');
+  return '[taskcluster ' + ts + '] ' + util.format.apply(this, args) + '\r\n';
 }
 
+/**
+ * Create a log message that will indicate that there was an error.  This log message
+ * will also include a line with timestamp useful for humans reading the logs and determining
+ * a timing of events.
+ *
+ * Message will always include a newline before the error string, so that
+ * it is at the beginning of the line where treeherder expects it, even if
+ * the last output was not newline-terminated.
+ */
 export function fmtErrorLog() {
   let args = Array.prototype.slice.call(arguments);
-  // always include a newline before this string, so that it is at the beginning of the line
-  // where treeherder expects it, even if the last output was not newline-terminated
-  return '\r\n[taskcluster:error] ' + util.format.apply(this, args) + '\r\n';
+  let msg = fmtLog(...args);
+  return msg + '[taskcluster:error] ' + util.format.apply(this, args) + '\r\n';
```

Comment:
I kinda don't think this duplication is worthwhile -- I suspect we don't want statistics on how long it takes for error XYZ to occur, so I don't think the timestamp is especially useful here.  If I'm wrong and it is useful, then this is a good approach.